CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+### Asking Questions
+
+For how-to questions and other non-issues, please use [StackOverflow](http://stackoverflow.com/questions/tagged/material-ui) or [Gitter](https://gitter.im/callemall/material-ui) chat instead of Github issues. There is a StackOverflow tag called "material-ui" that you can use to tag your questions.
+
+### Creating an Issue
+
+If you think you have found a bug, or have a new feature idea, please start by making sure it hasn't already been [reported or fixed](https://github.com/callemall/material-ui/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aclosed). You can search through existing issues and PRs to see if someone's reported one similar to yours.
+
+Next, create a new issue that briefly explains the problem, and provides a bit of background as to the circumstances that triggered it, and spteps to reproduce it. A minimal example for code issues, or images or animated gifs for design issues can help understand your issue.
```

Comment:
Would it be worth while to list some things that we general like to see in issues, maybe like these things?

bugs: browser, version, code example
feature request: screenshot/link to the spec